config/prod.exs

```diff
@@ -12,9 +12,9 @@ use Mix.Config
 # manifest is generated by the mix phoenix.digest task
 # which you typically run after static files are built.
 config :healthlocker, Healthlocker.Endpoint,
-  http: [port: {:system, "PORT"}],
-  url: [scheme: "http", host: System.get_env("HEROKU_URL") , port: 443],
-  force_ssl: [rewrite_on: [:x_forwarded_proto]],
+  http: [port: System.get_env("PORT")],
+  url: [scheme: "http", host: System.get_env("HEROKU_URL")], # port: 443],
```

Comment:
Are we still going to be using the `"HEROKU_URL"` from the environment variables now that we are no longer using heroku?